xAPI-Data.md

```diff
@@ -970,7 +970,7 @@ Run-Time Environment. See [Appendix C](#Appendix2C) for examples of each format.
 		<td>numeric</td>
 		<td>A range of numbers represented by a minimum and a maximum delimited by <code>[:]</code>. 
 			Where the range does not have a maximum or does not have a minimum, that number is omitted but the delimiter is
-			still used. E.g. ```"[:]4"``` indicates a maximum for 4 and no minimum. 
```

Comment:
This is probably an existing issue, but since this is inside a table it needs to use "&lt;code&gt;&lt;/code&gt;". Can we fix in this PR?